back-end/hub-core/src/main/java/io/apicurio/hub/core/integrations/EventSink.java

```diff
@@ -0,0 +1,15 @@
+package io.apicurio.hub.core.integrations;
+
+/**
+ * @author Fabian Martinez
+ */
+public interface EventSink {
+
+    String name();
+
+    boolean isConfigured();
+
+    void handle(ApicurioEventType type, String designId, byte[] data);
```

Comment:
I imagine we might eventually have events that are associated with an API design.  But that's ok - let's keep the `designId` param here for now and we can either modify or add another `handle()` variant in the future **if we need it**. :)